### PR TITLE
BETA: Register subzero.is-a.dev

### PR DESCRIPTION
**Pull request by TheODDYSEY**
Added `subzero.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/sa.bila.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheODDYSEY",
+        "email": "mosessabz07@gmail.com"
+    },
+    "record": {
+        "URL": "sa.bila.is.a.dev"
```
Comment (Stef-00012) [inline domains/sa.bila.json:7]: Invalid URL record (no protocol and would cause a redirect loop)

File: domains/sabila.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheODDYSEY",
+        "email": "mosessabz07@gmail.com"
+    },
+    "record": {
+        "URL": "sabila.is-a.dev"
```
Comment (Stef-00012) [inline domains/sabila.json:7]: Invalid URL record (no protocol and would cause a redirect loop)

File: domains/sabilamoses.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheODDYSEY",
+        "email": "mosessabz07@gmail.com"
+    },
+    "record": {
+        "URL": "sabilamoses.is.a.dev"
```
Comment (Stef-00012) [inline domains/sabilamoses.json:7]: Invalid URL record (no protocol and would cause a redirect loop)

File: domains/subzero.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheODDYSEY",
+        "email": "mosessabz07@gmail.com"
+    },
+    "record": {
+        "URL": "subzero.is.a.dev"
```
Comment (Stef-00012) [inline domains/subzero.json:7]: Invalid URL record (no protocol and would cause a redirect loop)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**